list/list.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
```

Comment:
It would be better if you put script tag in body and use defer attribute too

Reply:
This Script is for font-awesome..
And i want to load all the icon to load as soon the html rendering start..
That why i put it in head..